packages/sanes-chrome-extension/src/logic/blockchain/wallet.ts

```diff
@@ -4,6 +4,36 @@ import { ValueAndUpdates } from '@iov/stream';
 import { WalletInfo, HdPaths } from '@iov/keycontrol';
 import { Slip10RawIndex } from '@iov/crypto';
 
+export type Algorithm = 'secp256k1' | 'ed25519';
```

Comment:
This will soon bring a collision of the symbol `Algorithm`. Why not reuse the `Algorithm` enum from @iov/bcp?